tests/test_instance_settings_handler.cpp

```diff
@@ -504,6 +512,54 @@ TEST_F(TestInstanceSettingsHandler, setAddsInterface)
     EXPECT_EQ(specs[target_instance_name].extra_interfaces[1].mac_address, "");
 }
 
+TEST_F(TestInstanceSettingsHandler, setThrowsIfBridgingWrongInterface)
+{
+    bridged_interface = "wrong";
+
+    constexpr auto target_instance_name = "pappo";
+    specs.insert({{"blues", {}}, {"local", {}}, {target_instance_name, {}}});
+
+    mock_vm(target_instance_name); // TODO: make this an expectation.
+
+    std::string failure{"Invalid network 'wrong' set as bridged interface"};
+    MP_EXPECT_THROW_THAT(make_handler().set(make_key(target_instance_name, "bridged"), "true"),
+                         std::runtime_error,
+                         mpt::match_what(HasSubstr(failure)));
+
+    bridged_interface = "eth8";
+}
+
+TEST_F(TestInstanceSettingsHandler, setWithoutAuthorizationThrows)
+{
+    user_authorized = false;
+    constexpr auto target_instance_name = "pappo";
+    specs.insert({{"blues", {}}, {"local", {}}, {target_instance_name, {}}});
+
+    mock_vm(target_instance_name); // TODO: make this an expectation.
+
+    std::string failure{
+        "Cannot update instance settings; instance: pappo; reason: Need user authorization to bridge eth8"};
+    MP_EXPECT_THROW_THAT(make_handler().set(make_key(target_instance_name, "bridged"), "true"),
+                         mp::NonAuthorizedBridgeSettingsException,
+                         mpt::match_what(HasSubstr(failure)));
+
+    EXPECT_EQ(specs[target_instance_name].extra_interfaces.size(), 0u);
+    user_authorized = true;
```

Comment:
Again, not sure why this is at the end of the test?

Reply:
As above, I just wanted to set `user_authorized` to its previous value, but again this is not needed. Removed.